src/woo_search/search_index/index.py

```diff
@@ -16,14 +16,72 @@
 from .typing import IndexName, InformatieCategorieType, PublisherType
 
 
+def dutch_analyzer():
+    return {
+        "filter": {
+            "dutch_stemmer": {
+                "type": "stemmer",
+                "language": "dutch",
+            },
+            "dutch_stop": {
+                "type": "stop",
+                "stopwords": "_dutch_",
+            },
+        },
+        "analyzer": {
+            "dutch_analyzer": {
+                "type": "custom",
+                "filters": [
+                    "lowercase",
+                    "dutch_stop",
+                    "dutch_stemmer",
+                ],
+                "tokenizer": "standard",
+            }
+        },
+    }
+
+
+class Attachment(InnerDoc):
+    content: M[str] = mapped_field(
+        Text(
+            fields={"keyword": Keyword()},
+            analyzer="dutch_analyzer",
+            required=False,
+        )
+    )
+
+
 class Publisher(InnerDoc):
-    uuid: M[str] = mapped_field(Text(required=True, fields={"keyword": Keyword()}))
-    naam: M[str] = mapped_field(Text(required=True, fields={"keyword": Keyword()}))
+    uuid: M[str] = mapped_field(
+        Text(
+            fields={"keyword": Keyword()},
+            required=True,
+        )
+    )
+    naam: M[str] = mapped_field(
+        Text(
+            fields={"keyword": Keyword()},
+            analyzer="dutch_analyzer",
+            required=True,
+        )
+    )
```

Comment:
we're not doing any queries on these fields (at the moment), is is worth the effort of specifying the analyzer here?